File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/UploadBundleToInternalAppSharingTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.GooglePlayPublisher;


public class UploadBundleToInternalAppSharingTask extends BaseTask {

	public UploadBundleToInternalAppSharingTask() {
		setDescription("Upload Bundle to Internal App Sharing");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.uploadBundleToInternalAppSharing(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/UploadBundleToInternalAppSharingTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

class UploadBundleToInternalAppSharingTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().uploadBundleToInternalAppSharing(app)
    }

    init {
        description = "Upload Bundle to Internal App Sharing"
    }
}
